GitUI/BuildServerIntegration/BuildServerWatcher.cs

```diff
@@ -93,6 +93,7 @@ public void CancelBuildStatusFetchOperation()
             }
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times")]
```

Comment:
@mnaoumov One option is adding `Justification = "http://stackoverflow.com/questions/1065168/does-disposing-streamreader-close-the-stream"` to the `SuppressMessage` attribute.